spark/src/main/antlr4/io/delta/sql/parser/DeltaSqlBase.g4

```diff
@@ -88,6 +88,8 @@ statement
         DROP CONSTRAINT (IF EXISTS)? name=identifier                    #dropTableConstraint
     | ALTER TABLE table=qualifiedName
         DROP FEATURE featureName=featureNameValue (TRUNCATE HISTORY)?   #alterTableDropFeature
+    | ALTER TABLE table=qualifiedName
+        (clusterBySpec | CLUSTER BY NONE)                               #alterTableClusterBy
```

Comment:
FYI, we will add this to OSS Spark but this is a stop gap before the next Spark release.